### ISSUE TITLE
Visualize tree structure

### ISSUE THREAD

**Issue by oyvindbra** [httpie/http-prompt#206]
I'd like to be able to visualize the tree structure for the api endpoints

**Comment by BoboTiG** [comment]
Something like in #204?

**Comment by oyvindbra** [comment]
> 
> 
> Something like in #204?

Exactly. Just thought I'd post a request for it as well, to see if anyone else might be interested in the feature.
